.github/workflows/dynamo_bin_diff.yml

```diff
@@ -59,9 +58,8 @@ jobs:
       - name: Setup dotnet
         uses: actions/setup-dotnet@v5
         with:
-          dotnet-version: |
-            8.0.x
-            10.0.100-preview.5.25277.114
+          dotnet-version: 8.0.x
```

Comment:
What's the difference between specifying the dotnet version here and specifying it in global.json? Why is it 8.0.x here and .net 10 in global.json?

Reply:
- By referring global.json here, we can put dotnet-version on one place i.e. global.json. Next time we don't have to update in multiple places.
- We still need 8.0.x for DynamoServices https://github.com/DynamoDS/Dynamo/blob/68e90b159632e99864b492437592c3e68b04ccb1/src/NodeServices/DynamoServices.csproj#L19

Reply:
The last time Mike, Craig, and I were discussing this, we all had the same question. Why have net8 and net10 in addition to netstd2.0 for DynamoServices?